src/client/src/services/model/currencyPairPosition.js

```diff
@@ -1,10 +1,12 @@
 export default class CurrencyPairPosition {
   _symbol:string;
+  _ccy:string;
   _basePnl:number;
   _baseTradedAmount:number;
 
   constructor(symbol:string, basePnl:number, baseTradedAmount:number) {
     this._symbol = symbol;
+    this._ccy = symbol.length === 6 ? symbol.substr(0, 3) : '';
```

Comment:
Another note on this. The the service layer has a `CurrencyPair` class and this I believe has base and terms properties already, `CurrencyPairPosition` should use the `CurrencyPair` from the service layer not a string symbol